README.md

```diff
@@ -85,6 +64,23 @@ If the migration is complex or ambiguous, the migration script might have to be
 tweaked manually.
 
 
+## Local Testing
+
+Migrations are not used for local testing with SQLite. To setup local testing, simply define the `DATABASE_URL` environment variable
+
+```bash
+export DATABASE_URL=sqlite:///`pwd`/database.sqlite
+```
+and then invoke the tests
+
+```bash
+pytest -vx
+```
+
+A new SQLite database file will be automatically created for each test session and populated with tables based on the `pangeo_forge_orchestrator.models` module.
```

Comment:
Not sure if this true. I think it will just use whatever is at `DATABASE_URL`.

Reply:
Good catch, I think this should be "created if nonexistent, or updated to reflect new schema, if it already exists". Will update now.

Reply:
https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/34/commits/56457331323b1068868cb02b4ef2e0b2f730f9e7